cmd/minikube/cmd/console.go

```diff
@@ -47,14 +52,24 @@ var consoleCmd = &cobra.Command{
 		}
 		if consoleURLMode {
 			fmt.Fprintln(os.Stdout, url)
+		} else if machineReadAble{
+			host, port := getHostAndPort(url)
+			machineDetails = fmt.Sprintf(machineDetails, host, port, url)
+			fmt.Fprintln(os.Stdout, machineDetails)
 		} else {
 			fmt.Fprintln(os.Stdout, "Opening the OpenShift Web console in the default browser...")
 			browser.OpenURL(url)
 		}
 	},
 }
 
+func getHostAndPort(consoleUrl string) (string, string) {
+	return strings.Join(strings.Split(consoleUrl, ":")[:2], ":"),
```

Comment:
Personally I feel this is a bit cryptic. I'd suggest to add some comments on what you do or write this in a bit more self descriptive way. 

Also, this seems to fall short on what we defined in the issue. The host should just be the IP, not a URL.